tests/wicked/advanced_sut.pm

```diff
@@ -36,29 +53,33 @@ sub run {
     assert_script_run("cat $gre_config");
     assert_script_run('ifup gre1');
     assert_script_run('ip a');
-    my $ret = $self->ping_with_timeout(ip => "$parallel_ip_in_tunnel", timeout => '60');
+    $results{Test1} = get_test_result($ip_in_tunnel, "");
```

Comment:
$results{Test1} = get_test_result($self, $ip_in_tunnel, "");

Reply:
not important  now, but shall we keep the keys in lower case? i.e. $results{test1}, $results{test2}... or just numbers, $results{1}, $results{2}, 
This way, we can always numbers to some text and be more flexible when presenting the results.